internal/consensus/mempool_test.go

```diff
@@ -90,21 +100,30 @@ func TestMempoolProgressInHigherRound(t *testing.T) {
 		}
 		return cs.defaultSetProposal(proposal)
 	}
+
+	// Start test round
+	height, round := cs.Height, cs.Round
 	startTestRound(cs, height, round)
 
-	ensureNewRound(newRoundCh, height, round) // first round at first height
-	ensureNewEventOnChannel(newBlockCh)       // first block gets committed
+	// Ensure new round at first height and block commit
+	ensureNewRound(newRoundCh, height, round)
```

Comment:
The problem here is that the new round event takes more than 200ms?

Reply:
These are both good questions that I had missed. I will review this tomorrow and thank you for your review